hw/ip/rom_ctrl/data/rom_ctrl_testplan.hjson

```diff
@@ -63,6 +63,29 @@
       milestone: V3
       tests: []
     }
+    {
+      name: unsuccessful_rom_chk
+      desc: '''
+            Corrupt the expected digest so that there is a mismatch between expected digest and 
+            KMAC digest. 
+
+            **Checks**:
+            - Check that pwrmgr_data_o.good is not asserted. 
+            '''
+      milestone: V3
+      tests: []
+    }
+    {
+      name: corrupt_sig_fatal_chk
+      desc: '''
+            corrupt integrity of signals like the select signal to addr mux.
+            
+            **Checks**:
+            - Check if fatal error is flagged.
```

Comment:
The other testpoints phrase it as "Check that...", which I think is probably better. We don't want to check whether it happened, note that it didn't, and then pass the test :-)